src/main/java/net/io/kino/service/ReservationService.java

```diff
@@ -0,0 +1,18 @@
+package net.io.kino.service;
+
+import net.io.kino.model.Order;
+import net.io.kino.model.PersonalDetails;
+import net.io.kino.model.Ticket;
+import net.io.kino.model.TicketType;
+
+import java.util.List;
+
+public interface ReservationService {
+    boolean createOrder(List<Ticket> tickets, PersonalDetails client);
+    boolean confirmOrder(Order order);
+    boolean cancelOrder(Order order);
+    boolean addTicketType(TicketType type);
+    boolean updateTicketTypeByID(Long id);
```

Comment:
Z nazwy nie wynika, że to jest zmiana stanu biletu

Reply:
Inna sprawa to czy będziemy potrzebować deaktywować typy biletów?
Chyba będą tylko ulgowy i normalny